extensions/mongoose/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@loopback/mongoose",
+  "version": "0.0.1",
+  "description": "@loopback/mongoose",
+  "keywords": [
+    "loopback-extension",
+    "loopback"
```

Comment:
Add `mongodb` and `mongoose` into `keywords`?

Reply:
Done!